exercises/exercise3.js

```diff
@@ -1,12 +1,419 @@
+      + '\nI\'m here to help you discover plant species in the area.';
+    },
+    // isReply: true, --------------- If I have this set to true, terminal throws 'Shell: ...' before it's response?
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Responding to why so script 1',
+    prompt: 'I don\'t know what\'s happening',
+    handler: function () {
+      return 'Okay, easy. Let me explain...' 
+      + '\nI\'m a Native Australian Plant IDing bot.'
+      + '\nI\'m here to help you discover local plant species.';
+    },
+    // isReply: true,
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Responding to why so script 2',
+    prompt: 'i\'m worried about corona',
+    handler: function (){
+      return 'Ohhh, I think that\'s a little above my pay grade...';
+      },
+    // isReply: true,
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Responding to why so script 3',
+    prompt: 'i\'m worried about climate change',
+    handler: function (){
+      return 'Ohhh, I think that\'s a little above my pay grade in afraid...';
+      },
+    // isReply: true,
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Can you help me script',
+    prompt: 'Can you help me with #{topic}?',
+    handler: function(params) {
+      return 'Why don\'t you just Google ' 
+      + params.topic 
+      + '?' 
+      + ' ... it\'d be a lot faster'
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Not a helpful bot',
+    prompt: 'You\'re not very helpful',
+    handler: function() {
+      return 'I\'m not designed to solve *ALL* your problems...'
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Not a helpful bot',
+    prompt: 'Rude',
+    handler: function() {
+      return 'Yeah, well, I can only do what I do'
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Not a helpful bot',
+    prompt: 'That\'s a bit rude',
+    handler: function() {
+      return 'Look, I can\'t be everything'
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'What are you designed to do',
+    prompt: 'What are you designed to do?',
+    handler: function() {
+      var name = 'Dockrillia linguiformis';
+      var link = 'https://en.wikipedia.org/wiki/Dendrobium_linguiforme';
+      return 'I\'m designed to help you identify Native Australian plants...' 
+      + '\n Like the ' + name 
+      + ', aka Tongue Orchid, which is endemic to eastern Australia...'
+      + '\n' + link;
+    },
+        // -------- here i wanted to check if the user has already recieved the 'Responding to why so script 0 + 2' 
+    // so I could put and if else statement with one being the response currently given
+    // and the else being ' Like i mentioned... I'm deisgned ...' 
+    // but got a bit lost.. any tips?
```

Comment:
Yes! What a great idea to see if they've responded to a prior question. This is where Ken was talking about using State. State, you can think of as a very global variable that in this instance can be utilized by more than one of your prompt functions.
We can talk more about state and I'll make sure to send over some resource links for you. Basically you can store the responses a user gave to use later or even set a 'didAnswer' state to a boolean so it triggers true when a prompt is answered. Great looking ahead though!